modules/contracts/address-book.json

```diff
@@ -1179,135 +1179,7 @@
       "txHash": "0x1e787801c045702c8b8d6bcfea5ec0c3118dd088fb17e21e6248a730dbb6a572"
     },
     "Token": {
```

Comment:
We should probably remove the 1337 and 1338 entries from the address book completely since they are in .chaindata now.